cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -231,8 +231,10 @@ protected void configure() {
         setupClasspath();
 
         // Set local tmp dir to an absolute location in the twill runnable otherwise Hive complains
+        String localScratchPath = System.getProperty("java.io.tmpdir") + File.separator +
```

Comment:
This is the same as the current behavior for Hive 0.13, 0.14, and 1.1 (haven't checked other ones).